libs/cpa/data-api/src/lib/modules/workshops/workshops.service.spec.ts

```diff
@@ -6,19 +6,78 @@ import { Repository } from 'typeorm';
 import { Workshop } from '@tamu-gisc/cpa/common/entities';
 
 import { WorkshopsService } from './workshops.service';
+import { throwError } from 'rxjs';
```

Comment:
Wrong error. 

The error throw by TypeORM I think is a generic error.

Reply:
Deleted